### PR TITLE
Feature/transaction details page v3

### PR DESCRIPTION
**Pull request by predrag-babic**

### CODE REVIEW

File: pages/transactions/[id].js [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+            handleEditTransaction(updatedTransaction);
+            closeModal();
+          }}
+          variant="edit"
+        />
+      </Modal>
+
+      {successMessage && (
+        <StyledSuccessMessage>{successMessage}</StyledSuccessMessage>
+      )}
```
Comment (F-Kirchhoff) [inline pages/transactions/[id].js:102]: you should think about moving the success message into a "layout" component which is rendered on all pages. This way you don't have to include this code in every page (and might forget it on one).

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Money Manager Team,
what a huge PR 🥵. The most of it looks great, I added some suggestions to simplify you code a bit here and there. Please take a look.

File: pages/transactions/[id].js (hunk 1 above)
Comment (E-LeeSchaffer) [reply pages/transactions/[id].js:102]: Thank you, we would prefer to include to layout on a later stage.

**Review by JessicaLoers — APPROVED** [review]
Well done.  ✌️

A small note on the use of the `useEffect` hook, which I have already mentioned. But you can take a closer look at this in future a refactor story.

**Pull request merged**